sample-data/alexander/README.md

```diff
@@ -0,0 +1,23 @@
+Block 0 and Block 1 from PoC3 network.e
+
+For test purpose only. 
+
+Note that the Core_import_block should fail on importing block #1 because the stateRoot is computed using the old trie spec in PoC3 unless the RE API provides the old storage trie data to the runtime.
+
+block-001.hex contains data from the network message propagating the block. It is of the following struct type encoded using SCALED codec.
+
+    pub struct BlockData<Header, Hash, Extrinsic> {
+		/// Block header hash.
+		pub hash: Hash,
+		/// Block header if requested.
+		pub header: Option<Header>,
```

Comment:
We probably need to define `Header` as well? (Extrinsic below, we could just replace with `Vec<u8>`, since it is equivalent)